indico/modules/designer/controllers.py

```diff
@@ -177,8 +179,21 @@ def _process_args(self):
 
     def _process(self):
         title = f'{self.template.title} (copy)'
-        new_template = DesignerTemplate(title=title, type=self.template.type, data=self.template.data,
-                                        **self.target_dict)
+        new_template = DesignerTemplate(title=title, type=self.template.type, **self.target_dict)
+
+        data = self.template.data
```

Comment:
Careful! you end up mutating the original template's data here. I think the only reason this may not get saved is that SQLAlchemy doesn't realize it has been mutated...